docs/docusaurus/docs/cloud/expectations/manage_expectations.md

```diff
@@ -24,31 +24,52 @@ To create custom SQL query Expectations, you'll need to use the GX API. See [Cus
+| Cardinality        | `expect_select_column_values_to_be_unique_within_record`  | Expect the values for each record to be unique across the columns listed. Note that records can be duplicated.                         |
+| Cardinality        | `expect_column_proportion_of_unique_values_to_be_between` | Expect the proportion of unique values to be between a minimum value and a maximum value.                                              |
+| Cardinality        | `expect_column_unique_value_count_to_be_between`          | Expect the number of unique values to be between a minimum value and a maximum value.                                                  |
+| Data Integrity     | `expect_column_pair_values_to_be_equal`                   | Expect the values in column A to be the same as column B.                                                                              |
+| Data Integrity     | `expect_multicolumn_sum_to_equal`                         | Expect that the sum of row values in a specified column list is the same for each row, and equal to a specified sum total.             |
+| Distribution       | `expect_column_pair_values_A_to_be_greater_than_B`        | Expect the values in column A to be greater than column B.                                                                             |
+| Distribution       | `expect_column_values_to_be_between`                      | Expect the column entries to be between a minimum value and a maximum value (inclusive).                                               |
+| Distribution       | `expect_column_z_scores_to_be_less_than`                  | Expect the Z-scores of a column's values to be less than a given threshold.                                                            |
+| Distribution       | `expect_column_stdev_to_be_between`                       | Expect the column standard deviation to be between a minimum value and a maximum value.                                                |
+| Distribution       | `expect_column_sum_to_be_between`                         | Expect the column to sum to be between a minimum value and a maximum value.                                                            |
+| Missingness        | `expect_column_values_to_be_null`                         | Expect the column values to be null.                                                                                                   |
+| Missingness        | `expect_column_values_to_not_be_null`                     | Expect the column values to not be null.                                                                                               |
+| Numerical Data     | `expect_column_max_to_be_between`                         | Expect the column maximum to be between a minimum and a maximum value.                                                                 |
```

Comment:
You've indicated inclusivity in other places where a min/max range is expected -- is this also inclusive of the min/max values of the range?  If so we should indicate that.  If not, we should probably say (not inclusive) so that people don't have to guess.  Both files should be updated.

Reply:
the inclusivity is a parameter ("strict")